src/images/PalmSharp.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { ViewStyle } from 'react-native'
+import Svg, { Path } from 'react-native-svg'
+
+interface Props {
+  width?: number
+  height?: number
+  color?: string
+  style?: ViewStyle
+}
+
+export default function PalmSharp({ width = 296, height = 177, style }: Props) {
```

Comment:
@kathaypacific I didn't know what to call it cause they both are basically palms, one just having sharp leaves being the only difference. So that's where I come from in terms of the naming considering there was already "Palm" used.

I can call it a "Leaf" if that's preferable.